app/models/data_mapper_setup.rb

```diff
@@ -0,0 +1,22 @@
+# require 'data_mapper'
+# require 'dm-postgres-adapter'
+#
+# # require_relative './models/'
+# # require_relative './models/'
+# # require_relative './models/'
+#
+# DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/bookmark_manager_#{ENV['RACK_ENV']}")
+# DataMapper.finalize
```

Comment:
Commented out code can make it hard to read later - consider removing code if no longer needed before opening a pull request, or checking out a different branch if you are experimenting and want to come back to it later.